image.go

```diff
@@ -34,6 +34,7 @@ func ParseImageID(s string) (ImageID, error) {
 		return ImageID{}, ErrBlankImageID
 	}
 	var img ImageID
+	s = strings.Split(s, "@")[0]
```

Comment:
I guess this is to do with image digests -- we should address this separately (since Kubernetes also accepts digests).